composer.json

```diff
@@ -25,5 +26,8 @@
         "psr-4": {
             "MailerLiteApi\\Tests\\": "tests/"
         }
+    },
+    "conflict": {
+        "php-http/httplug": "< 2"
```

Comment:
adding this conflict to be sure to not break for users who provide their own httplug client implementation. the httplug 2.* interface extends the psr-18 interface to be compatible with both.